src/server/iframe.html.js

```diff
@@ -8,34 +8,43 @@ import url from 'url';
 //   'preview.0d2d3d845f78399fd6d5e859daa152a9.css',
 //   'static/preview.9adbb5ef965106be1cc3.bundle.js.map',
 //   'preview.0d2d3d845f78399fd6d5e859daa152a9.css.map' ]
-const previewUrlsFromAssets = (assets) => {
+const urlsFromAssets = (assets) => {
   if (!assets) {
```

Comment:
Do you think you could refactor this into a single return statement? I think this function has become too complex.